eventhandler/__init__.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, before_handler=None, error_on_missing_type=True, ignore_handl
         self.ignore_handler_exceptions = ignore_handler_exceptions
 
         logger.debug("Registered the following event handlers:")
-        for event, handlers in self.handlers.iteritems():
```

Comment:
I think it would've been fine to convert this to `items`, as the dict of handlers is never huge, but doesn't matter too much.

Reply:
Oh I didn't actually test if that would work, I assumed it wouldn't. Looks like it does though, but it might be a bit of performance hit (though that's probably what your remark about the low amount of handlers was about :)). I think I'll keep it like this, I kind of like it :)